kokoro/config/test/common.gcl

```diff
@@ -15,7 +15,15 @@ template config common = {
       // The new Cloud NAT gateway must have "Minimum ports per VM instance"
       // set to something >160 as per this article:
       // https://www.suse.com/support/kb/doc/?id=000019662
-      ZONE = 'us-central1-b'
+      ZONES =join([
+        'us-central1-a=3',
+        'us-central1-b=3',
+        'us-central1-c=3',
+        'us-central1-f=3',
+        'us-east1-b=2',
+        'us-east1-c=2',
+        'us-east1-d=2',
+      ], ',')
```

Comment:
Any reason we don't add a function in `utils/functions.gcl`, like:

```
// Convert a map into a comma-separated list of key-value pairs.
//
// Example usage:
//  make_zones(['foo': 'bar', 'x': 'y'])
// Returns
//  'foo=bar,x=y'
make_zones = lambda zones_: {
  assert is_map(zones_)
  ret = join(map(lambda x_: "%x_[0]%=%x_[1]%", zones_.items()), ',')
}.ret
```
and use it here:
```suggestion
      ZONES = functions.make_zones([
        'us-central1-a': 3,
        'us-central1-b': 3,
        'us-central1-c': 3,
        'us-central1-f': 3,
        'us-east1-b': 2,
        'us-east1-c': 2,
        'us-east1-d': 2,
      ])
```
?

Reply:
well I don't see the point in doing that. It's adding an extra layer of indirection and a helper function with some actual logic in it, and I don't think the result is any better.

Reply:
Making it a map guarantees that the keys are unique, and removes the possibility of, e.g., stray spaces screwing up the flag. But up to you.

Reply:
I still think I'd rather not add that layer of indirection just for that.